package/src/renderer/processors/Animations/Animations.ts

```diff
@@ -24,13 +44,15 @@ export const materialize = <T>(
   const result = { ...props };
   mapKeys(props).forEach((key) => {
     const value = props[key];
-    if (isAnimatedValue(value)) {
+    if (isPropCallback(value)) {
       result[key] = value(ctx);
+    } else if (isAnimationValue(value)) {
+      result[key] = (value as unknown as ReadonlyValue<T[typeof key]>).value;
     }
   });
   return result as T;
 };
 
 export type AnimatedProps<T> = {
-  [K in keyof T]: T[K] | ((ctx: DrawingContext) => T[K]);
+  [K in keyof T]: T[K] | ((ctx: DrawingContext) => T[K]) | ReadonlyValue<T[K]>;
```

Comment:
Should we remove the ((ctx: DrawingContext) => T[K])? In addition to renaming `ReadonlyValue` to something more explicit would be the type annotations look very crisp and self descriptive.